app/build.gradle

```diff
@@ -265,7 +265,7 @@ android.applicationVariants.all { variant ->
 // -------------------------------------------------------------------------------------------------
     // NB: flipping SEND_TAB_ENABLED flag back and worth is currently not well supported and may need hand-holding.
     // Consult with the android-components peers before changing.
-    buildConfigField 'Boolean', 'SEND_TAB_ENABLED', (buildType == "nightly" || isDebug).toString()
+    buildConfigField 'Boolean', 'SEND_TAB_ENABLED', (buildType == "nightly" || buildType == "nightlyLegacy" || isDebug).toString()
```

Comment:
Since those constants (`buildType` etc.) are also defined in `BuildConfig` I wonder why you go through that pain and dynamically build your constants here. Wouldn't it be easier to construct them in Kotlin code in a feature flags class?

Reply:
I was initially following the process for this from another feature flag that was here, so I assumed it was supposed to keep all the flags in one place. I can look into replacing this.

Reply:
I'm going to change this in a follow up that I've [logged here](https://github.com/mozilla-mobile/fenix/issues/4075) since it's a bigger change for this issue.